src/chrome/content/rules/sfr.fr.xml

```diff
@@ -0,0 +1,34 @@
+<ruleset name="SFR (Partial)">
+    <target host="*.s-sfr.fr"/>
+    <target host="accessoires.sfr.fr"/>
+    <target host="adsl.sfr.fr"/>
+    <target host="applis.sfr.pro"/>
+    <target host="assistance.sfr.fr"/>
+    <target host="boutique.home.sfr.fr"/>
+    <target host="box.sfr.fr"/>
+    <target host="cloud.sfr.fr">
+    <target host="cms.sfr.fr"/>
+    <target host="cms-admin.sfr.fr"/>
+    <target host="contacteznous.sfr.fr"/>
+    <target host="elr.sfr.fr"/>
+    <target host="espace-client.sfr.fr"/>
+    <target host="etoolonline.sfr.fr"/>
+    <target host="forum.sfr.fr"/>
+    <target host="home.sfr.fr"/>
+    <target host="hotspot.wifi.sfr.fr"/>
+    <target host="internet3gplus.sfr.fr"/>
+    <target host="kara.sfr.fr"/>
+    <target host="m.sfr.f"/>
+    <target host="messagerie.sfr.fr"/>
+    <target host="metrics.sfr.fr"/>
+    <target host="monprofil.sfr.fr"/>
+    <target host="neufbox.sfr.fr"/>
+    <target host="presse.sfr.fr"/>
+    <target host="red.sfr.fr"/>
+    <target host="smetrics.sfr.fr"/>
+    <target host="www.cloud.fr">
+    <target host="www.sfr.fr"/>
+
+    <rule from="^http://metrics.sfr.fr" to="https://smetrics.sfr.fr"/>
```

Comment:
you need to add (not replace) after line 32,
`<test url="http://smetrics.sfr.fr/" />`
(it's not https in this case)